michelin.py

```diff
@@ -30,25 +42,29 @@ def _get_rest_info(uri):
 	for field in FIELD:
 		info_pane = rest_info.findAll('dl', {'class': field})
 		if len(info_pane) > 0:
-			parsed_info[field] = info_pane[0].text
+			# Structure changed, now the dl has dt and dd. We want the dd only.
+			parsed_info[field] = info_pane[0].find('dd').get_text()
 		else:
 			parsed_info[field] = 'n/a'
 	return parsed_info
 
 def scrape():
-	with open('japan_mich_rest.csv', 'wb') as f:
+	with open(FILE_NAME, 'w') as f:
 		w = csv.writer(f)
 		header_written = False
-		for i in range(1, 52):
-			for place in _get_index(page=i):
+
+		for page in range(1, NUM_PAGES+1):
+			for place in _get_index(page=page):
 				rest_data = _get_rest_info(place)
 				
 				if not header_written:
 					w.writerow(rest_data.keys())
 					header_written = True
+
 				vals = rest_data.values()
-				w.writerow([v.encode('utf-8') for v in vals])
```

Comment:
I think we still need to encode to `utf-8` here?

Reply:
Hmm it's working without the encoding to utf-8.
I removed the binary writing, maybe that makes it not necessary?
Happy to add it back if you want (and I can figure out how to python)

Reply:
I think we need it just in case there are some strange Kanji characters.

Reply:
If I writerow with `[v.encode('utf-8') for v in vals]`, every cell in the resulting csv is wrapped in `b'...'`.

If I instead open the file with these extra params, it outputs without the `b'...'`
`with open(FILE_NAME, 'w', newline='', encoding='utf-8') as f:`

Lemme know which one you want me to go with and I'll push those changes.

Reply:
https://stackoverflow.com/a/37364412

Reply:
Either is fine :)

Thanks for looking into it.